packages/clerk-js/src/ui/elements/SocialButtons.tsx

```diff
@@ -246,6 +268,22 @@ const SocialButtonBlock = forwardRef((props: SocialButtonProps, ref: Ref<HTMLBut
         props.sx,
       ]}
     >
+      {lastAuthenticationStrategy && (
+        <Box
+          elementDescriptor={descriptors.lastAuthenticationStrategyBadgeContainer}
+          sx={t => ({
+            position: 'absolute',
+            top: '-40%',
+            right: `calc(${t.space.$2x5} * -1)`,
+            borderRadius: t.radii.$lg,
+            backgroundColor: t.colors.$colorBackground,
+            padding: t.space.$px,
+          })}
+        >
```

Comment:
do we need this container vs applying the positioning directly to the badge component? is it just to support the px border? Could that be a outline around the badge?

Reply:
if we need to keep the box, we should render it as a span, as divs are not valid child elements of buttons

Reply:
The border radius from the background doesn't follow the radius of the inner badge 
<img width="590" height="230" alt="Screenshot 2025-09-05 at 6 21 30 PM" src="https://github.com/user-attachments/assets/5227c116-3d73-45c7-9181-7b510e9ecc1e" />

Reply:
@alexcarpenter Updated to use a single element. Had to hack around using pseudo elements, etc. to make things work given the alpha scales.

Reply:
<img width="764" height="294" alt="Screenshot 2025-09-05 at 11 04 24 PM" src="https://github.com/user-attachments/assets/bfcf6be6-67fe-48c1-9f32-518f9fc96599" />